src/main/java/com/parkinglot/parkinglotfees/utils/ParkingUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.parkinglot.parkinglotfees.utils;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class ParkingUtils {
+
+    public static String getFormattedDate(LocalDateTime currentDateTime){
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MMM-yyyy HH:mm:ss");
```

Comment:
better to pass the formatting as a parameter, this way this function can be reused for any other type of formatting

Reply:
done